config/routes.rb

```diff
@@ -1,3 +1,3 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  resources :users, only: %i[index destroy]
```

Comment:
Can we create/update the users?

Reply:
@okohuch Devise will handle that functionality and at that moment of time, it was not included into a project.

Reply:
resolve conflict with develop branch